modules/kogito-trusty-postgresql/module.yaml

```
@@ -0,0 +1,12 @@
+schema_version: 1
+name: org.kie.kogito.trusty.postgresql
+version: "2.0.0-snapshot"
+
+artifacts:
+  - name: trusty-service-postgresql-runner.jar
+    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/trusty-service-postgresql/2.0.0-SNAPSHOT/trusty-service-postgresql-2.0.0-20211005.110152-1.jar
+    md5: c9310c2fbdd0a20e6a14afea2acb6415
```

Just noticed it should be the runner jar ...

```suggestion
    url: https://repository.jboss.org/nexus/content/groups/public/org/kie/kogito/trusty-service-postgresql/2.0.0-SNAPSHOT/trusty-service-postgresql-2.0.0-20211005.110152-1-runner.jar
    md5: 40991d7f2a95cf313bb973b7ccbe85c8
```